src/webextension/injections/js/bug1457335-histography.io-ua-change.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+/**
+ * Bug 1457335 - histography.io - Override UA & navigator.vendor
+ * WebCompat issue #1804 - https://webcompat.com/issues/1804
+ *
+ * This site is using a strict matching of navigator.userAgent and
+ * navigator.vendor to allow access for Safari or Chrome. Here, we set the
+ * values appropriately so we get recognized as Chrome.
+ */
+
+/* globals exportFunction */
+
+const DEFAULT_UA = navigator.userAgent;
+
+Object.defineProperty(window.wrappedJSObject.navigator, "userAgent", {
+  get: exportFunction(function() {
+    return DEFAULT_UA + " Chrome for WebCompat";
+  }, window),
+
+  set: exportFunction(function() {}, window)
+});
+
+Object.defineProperty(window.wrappedJSObject.navigator, "vendor", {
+  get: exportFunction(function() {
```

Comment:
Tiniest of nits here, but I Chrome uses "Google Inc." with the period at the end. I suspect it doesn't matter at all, but I thought I'd mention it.